mobile-widgets/qml/CloudCredentials.qml

```diff
@@ -125,6 +125,7 @@ Item {
 				id: signin_register_normal
 				text: qsTr("Sign-in or Register")
 				onClicked: {
+					PrefCloudStorage.cloud_auto_sync = true
```

Comment:
why do you want to enable auto_sync by default?
An unsuspecting user could create quite hefty data bills while roaming with this setting - that's why it's off by default and you need to manually turn it on.

Reply:
Apologies, the curse of knowledge has hit me when writing the commit message.
The actual relevant bit of change is this here:

https://github.com/subsurface/subsurface/pull/4094/files#diff-40007045ffbbfbaeff5d836fd0b0974f15574ea83cfb11783da2837aed84f63dR354

I.e. in the existing version `cloud_auto_sync == true`  means 'cloud auto synch is disabled', which is likely the root cause for the warning message (https://github.com/subsurface/subsurface/pull/4094/files#diff-40007045ffbbfbaeff5d836fd0b0974f15574ea83cfb11783da2837aed84f63dL356) being shown at the wrong time (when auto synch is enabled), which is what Martin had reported and what triggered my investigation.